wallet/storage/src/schema.rs

```diff
@@ -33,7 +33,7 @@ storage::decl_schema! {
         /// Store keychain usage states
         pub DBKeychainUsageStates: Map<AccountKeyPurposeId, KeychainUsageState>,
         /// Store for all the private keys in this wallet
-        pub DBRootKeys: Map<RootKeyId, RootKeyContent>,
+        pub DBRootKeys: Map<RootKeyId, Vec<u8>>,
```

Comment:
I'd still prefer some sort of wrapper type that communicates it's a (possibly encrypted) root key over a plain `Vec<u8>`.

Reply:
Created a wrapper struct `MaybeEncrypted<T>` in util